### PR TITLE
docs: updated 04-chapter-3-adding-mutations-to-your-api.mdx

### PR DESCRIPTION
**Pull request by canrozanes**
This is something I came across as I was going through the tutorial. Not explicitly setting an export key on contextType appears to throw a Typescript error:

`Property 'export' is missing in type '{ module: string; alias: string; }' but required in type 'TypingImport'.`

### CODE REVIEW

File: docs/content/010-getting-started/03-tutorial/04-chapter-3-adding-mutations-to-your-api.mdx [hunk 1]
```diff
@@ -131,6 +131,7 @@ export const schema = makeSchema({
 +  contextType: {                                    // 1
 +    module: join(__dirname, "./context.ts"),        // 2
 +    alias: "ContextModule",                         // 3
++    export: "Context"                               // 4
```
Comment (namoscato) [inline docs/content/010-getting-started/03-tutorial/04-chapter-3-adding-mutations-to-your-api.mdx:134]:
```suggestion
+    export: "Context",                              // 3
```

File: docs/content/010-getting-started/03-tutorial/04-chapter-3-adding-mutations-to-your-api.mdx [hunk 2]
```diff
@@ -154,6 +155,7 @@ export const schema = makeSchema({
   contextType: {                                    // 1
     module: join(__dirname, "./context.ts"),        // 2
     alias: "ContextModule",                         // 3
+    export: "Context"                               // 4
```
Comment (namoscato) [inline docs/content/010-getting-started/03-tutorial/04-chapter-3-adding-mutations-to-your-api.mdx:158]:
```suggestion
    export: "Context",                              // 3
```

File: docs/content/010-getting-started/03-tutorial/04-chapter-3-adding-mutations-to-your-api.mdx [hunk 3]
```diff
@@ -165,6 +167,7 @@ export const schema = makeSchema({
 1. Option to set the context type
 1. Path to the module where the context type is exported
 1. Name of the export in that module
+1. Name of the type generated in nexus-typegen
```
Comment (namoscato) [inline docs/content/010-getting-started/03-tutorial/04-chapter-3-adding-mutations-to-your-api.mdx:170]:
```suggestion
```

**Review by namoscato — COMMENTED** [review]
Thanks, @canrozanes!

I noticed this as well and was going to make this change, which removes the optional `alias` in favor of `export`. It might be a simpler option, unless we find it useful to introduce `alias` in the tutorial?

**Review by jasonkuhrt — CHANGES_REQUESTED** [review]
This may have already been fixed. Please rebase and see.

**Pull request closed without merging**